src/Symfony/Component/Serializer/composer.json

```diff
@@ -27,6 +27,9 @@
         "doctrine/annotations": "~1.0",
         "doctrine/cache": "~1.0"
     },
+    "conflict": {
+        "symfony/property-access": ">=3.0,<3.0.4|>=2.8,<2.8.4"
```

Comment:
Then I would rather update the constraint in `require-dev`to `~2.8,>=2.8.4|~3.0,>=3.0.4`.